pcm.version

```diff
@@ -0,0 +1 @@
+3.3.0
```

Comment:
suggestion: what about having this version in the `package.json` of the frontend?
It can be retrieved at release time with `npm pkg get version` and we avoid having a new file just for versioning

Reply:
This is a great idea!! Thanks
It'll be in the next revision!